src/components/QuizResults.tsx

```diff
@@ -15,7 +15,7 @@ import FacebookIcon from '@mui/icons-material/Facebook';
 import Link from './Link';
 import { HeadWithMeta } from './HeadWithMeta';
 import CircularProgress from '@mui/material/CircularProgress';
-import Backdrop from '@mui/material/Backdrop';
+import LoadingButton from '@mui/lab/LoadingButton'
```

Comment:
Didn't realize this was part of the "lab". I don't think it matters, but just weird that their docs don't mention it on the Button page.